diagnostic.md

```diff
@@ -20,6 +27,9 @@ c = c + d
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d` and explain what they do.
 
+value of c = 7
+perform operations on the right hand side of the equals sign and then assign that value to c on the left
+
```

Comment:
Yes specifically the operators were `=` which assigns value and `+` which adds two values.